packages/xo-server/src/api/host.js

```diff
@@ -314,3 +314,23 @@ scanPifs.params = {
 scanPifs.resolve = {
   host: ['id', 'host', 'administrate'],
 }
+
+// -------------------------------------------------------------------
+
+export async function installCertificate({ host, ...props }) {
+  host = this.getXapiObject(host)
+  await host.$xapi.installCertificateOnHost(host.$id, props)
+}
+
+installCertificate.description = 'Install a certificate on a host'
+
+installCertificate.params = {
+  id: { type: 'string' },
+  certificate: { type: 'string' },
+  certificateChain: { type: 'string', optional: true },
```

Comment:
IMHO `chain` is enough.

Reply:
It is understandable in the API? because the XAPI make it clear that it's the certificate chain.

Reply:
Yes, this method is about installing a certificate, it's as clear as `privateKey`.